ci/env/kava-protonet/genesis.json

```diff
@@ -3006,7 +3006,9 @@
       },
       "in_flight_packets": {}
     },
-    "precisebank": {},
+    "precisebank": {
+      "remainder": "0"
```

Comment:
i talked with you about this outside of github, but worth noting here:

the remainder should always be zero. if it is not that means we either didn't properly increment/decrement accounts with their owned balance or we increased the total supply. in either case, something is wrong.

we should definitely verify that in == out in memory, but it doesn't need to be chain state.

Reply:
documenting this discussion:
our expectation is for the remainder to be 0 and never change for our use case. however, that _doesn't_ mean that precisebank should not be prepared to handle a world in which there is a non-zero remainder (or one that changes).
the concern of it being 0 in our use of it in x/evm is a higher-level concern than x/precisebank should enforce.
we'll keep remainder in genesis state because it makes for a more pure & correct bank implementation 👍